src/v2/Apps/Collect/Routes/Collections/Components/CollectionsCategory.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react"
+import {
+  Box,
+  Button,
+  Column,
+  EntityHeader,
+  GridColumns,
+  Text,
+} from "@artsy/palette"
+import { crop } from "v2/Utils/resizer"
+import { createFragmentContainer, graphql } from "react-relay"
+import { CollectionsCategory_category } from "v2/__generated__/CollectionsCategory_category.graphql"
+
+interface CollectionsCategoryProps {
+  category: CollectionsCategory_category
+}
+
+export const CollectionsCategory: React.FC<CollectionsCategoryProps> = ({
+  category,
+  category: { collections },
+}) => {
+  const [showAll, toggleShowAll] = useState(false)
+
+  const sorted = [...collections].sort((a, b) => a.title.localeCompare(b.title))
+
+  const displayableCollections =
+    showAll || isServer ? sorted : sorted.slice(0, TRUNCATED_AMOUNT)
```

Comment:
I copied this logic over; I think this is trying to render all of them for Google but truncate them for the real user? It's kinda weird... leads to a flash if you do a full page load. I guess we could just hide them with CSS instead?

Reply:
I vaguely remember this, yeah it was for SEO reasons, a css solution would def be better but not a blocker

Reply:
71cf58143669271c1fa6c9df038212dac637a097 👍